cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -295,4 +327,126 @@ bool CobaltContentBrowserClient::WillCreateURLLoaderFactory(
   return true;
 }
 
+void CobaltContentBrowserClient::CreateExperimentConfig() {
+  if (!exp_config_) {
+    auto pref_registry = base::MakeRefCounted<PrefRegistrySimple>();
+
+    RegisterPrefs(pref_registry.get());
+
+    base::FilePath path;
+    CHECK(base::PathService::Get(content::SHELL_DIR_USER_DATA, &path));
+    path = path.Append(kExperimentConfigFilename);
+
+    PrefServiceFactory pref_service_factory;
+    pref_service_factory.set_user_prefs(
+        base::MakeRefCounted<JsonPrefStore>(path));
+
+    exp_config_ = pref_service_factory.Create(pref_registry);
+  }
+}
+
+void CobaltContentBrowserClient::CreateFeatureListAndFieldTrials() {
+  CreateExperimentConfig();
+  SetUpFieldTrials();
+}
+
+void CobaltContentBrowserClient::SetUpCobaltFeaturesAndParams(
+    std::unique_ptr<base::FeatureList>& feature_list) {
```

Comment:
It's uncommon to use a std::unique_ptr<>& in Chromium (`If the function does not modify t's ownership, declare the param as T*` https://chromium.googlesource.com/chromium/src/+/main/styleguide/c++/c++.md#object-ownership-and-calling-conventions), instead just use a naked pointer.